lib/Controllers/update.js

```diff
@@ -47,10 +28,45 @@ Update.prototype.write = function(req, res, context) {
       context.attributes[a] = req.params[a];
   });
 
+  // Update associated data
+  var self = this,
+      associationPrimKeys = [];
+  if (this.include && this.include.length) {
+    this.include.forEach(function(i) {
+      var instanceModel = i.model || i,
```

Comment:
I personally feel that this should be transformed when the resource is built, so that if the user provides something like `include: [SomeModel]`, epilogue will automatically change that to `include: [{ model: SomeModel }]`. We're going to be referencing includes more and more, and it would be nice to not have to do this check every time.